.travis.yml

```diff
@@ -85,8 +77,8 @@ jobs:
       env: MODULE='berkeleyje'
     - <<: *STANDARD_TEST_JOB
       env: MODULE='test'
-    - <<: *TINKERPOP_TEST_JOB
```

Comment:
Why tinkerpop test job is removed?

Reply:
I included tinkerpop test execution directly into the inmemory tests. 

If we add tinkerpop test execution for other backends, we can add this again.